test/is/datetime.test.js

```diff
@@ -0,0 +1,47 @@
+import is from '../../src/is/datetime';
+import { testFalsyWithNullable } from '../utils';
+
+describe('isPast', () => {
+    test('returns true on past date objects', () => {
+        expect(is.past(new Date(0))).toBeTruthy();
+        expect(is.past(new Date(1000))).toBeTruthy();
+        expect(is.past(new Date('April 30, 1993 16:40:00'))).toBeTruthy();
+    });
+    test('returns false on anything else', () => {
+        expect(is.past(new Date('November 23, 3998 03:24:00'))).toBeFalsy();
+        expect(is.past('November 23, 3998 03:24:00')).toBeFalsy();
+        expect(is.past(new Date())).toBeFalsy();
```

Comment:
Poorly designed.
Depends on the execution speed and the precision of the clock.
Please fix.